consoles/VNC.pm

```diff
@@ -738,7 +738,7 @@ sub map_and_send_key {
             next;
         }
         else {
-            die "No map for '$key'";
+            die "Key '$key' not found in map ($keys)";
```

Comment:
```suggestion
            die "No mapping for key '$key' available (full key sequence: $keys)";
```

Reply:
There's no documentation of expected input or errors so first of all I tried to add context to message on what *is* expected regardless of input. I like your improved version!

Reply:
To answer the question, `$keys` could be `"echo test"` or one out of minus/tab/res/backspace/esc.